### DISCUSSION TITLE
accessing data without removing the SD card

### DISCUSSION THREAD

**Discussion by Chomemel** [sanni/cartreader#682] [Need help]
Is there a way to access the data on the SD card while plugin the arduino my PC ?

The SD module is very sensitive (or is it sensible?). If I touch it just a tiny bit, then the CART READER won’t work because of an SD ERROR, and it takes forever to get rid of it.
So if I can access the data without removing the SD card from the module, it would be a HUGE help for me.

Thanks.

**Comment by Cephiros** [comment]
I would reflow, or replace the sd carrier. I dont think your request is possible in the current state.

**Comment by Ancyker** [comment]
If you can't find the issue, you could use a microSD card with an adapter instead and just insert/remove the microSD card from the adapter rather than the entire adapter, but if literally just touching it causes issues you need to fix that.

**Comment by Chomemel** [comment]
Trust me I tried a lot of things :
- I reflowed the pins
- I tried another SD module
- I tried another SD card
- I tested the continuity between pins from the module to the arduino. Checked it with power on, power off, while it was working, while it was written SD ERROR. And I had the same results with the error and without it

I don't really want to touch anything again. Because I soldered, dessoldered and resoldered lots a thing. And now I'm afraid to harm the whole thing to a way I won't be able to fix it.

That's why downloading the files from the SD card to my PC just by pluging the Cart Reader would be a HUGE HUGE help for (if it is possible of course).

**Comment by sanni** [comment]
Something like this could work: https://www.youtube.com/watch?v=nHNZPRl8gzA

**Comment by Chomemel** [comment]
I tried to remove and insert my micro sd card several times without an issue.  The 3D printed case seems to solve the problem a bit.

But next time it becomes picky, I'll move to the ultimate solution: soldering wires from the SD module pins to the arduino itself. Because I think my problem comes from the female pinheaders.
